service/rpc2/server.go

```diff
@@ -770,7 +770,8 @@ type ExamineMemoryIn struct {
 
 // ExaminedMemoryOut holds the return values of ExamineMemory
 type ExaminedMemoryOut struct {
-	Mem []byte
+	Mem            []byte
+	IsLittleEndian bool
```

Comment:
Maybe we can just store `binary.BigEndian` or `binary.LittleEndian` and just use the methods off of them when we need to convert, etc.

Reply:
I don't think those can be serialized to json.

Reply:
Of course not... obvious oversight on my part.